bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/types/BTupleType.java

```diff
@@ -135,11 +135,6 @@ public String toString() {
         return readonly ? toString + " & readonly" : toString;
     }
 
-    @Override
-    public String getName() {
-        return toString();
```

Comment:
@manuranga, @warunalakshitha, for certain types we seem to have returned the `toString()` representation as the name. If there is no type name, this would return the shape string. IMO, we can't consider the shape string as the type name, wdyt?

Reply:
+1. Shall we make this method final and remove the other override methods.

Reply:
There seems to be one particular case where we need to override in `BMethodType` - io.ballerina.runtime.internal.types.BMethodType#getName